### ISSUE TITLE
java.lang.ClassCastException when a Java interface file is available in the workspace

### ISSUE THREAD

**Issue by rafaeltuelho** [camel-tooling/camel-lsp-client-vscode#475]
# Issue information

> Define issue title **short** and **most specific** to the problem.

> Tag each new issue with relevant **LABELS** (*Bug*, *Improvement*, etc.).

### Environment

- Language Support for Apache Camel version: "**v0.0.28**"
- OS:
    - [ ] Linux (*RHEL*, *Fedora*, *Ubuntu*, ...)
    - [ ] Windows
    - [ x ] macOS

### Description of wrong behavior

- ...

### Exact steps to reproduce issue

1. ...
2. ...
3. ...

### Description
```
java.util.concurrent.CompletionException: java.lang.ClassCastException: class org.jboss.forge.roaster.model.impl.JavaInterfaceImpl cannot be cast to class org.jboss.forge.roaster.model.source.JavaClassSource (org.jboss.forge.roaster.model.impl.JavaInterfaceImpl and org.jboss.forge.roaster.model.source.JavaClassSource are in unnamed module of loader org.springframework.boot.loader.LaunchedURLClassLoader @5c7fa833)
```

### Expected behaviour

- ...

### Screenshot (*optional*)

```
Nov 16, 2020 3:02:31 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint fallbackResponseError
SEVERE: Internal error: java.lang.ClassCastException: class org.jboss.forge.roaster.model.impl.JavaInterfaceImpl cannot be cast to class org.jboss.forge.roaster.model.source.JavaClassSource (org.jboss.forge.roaster.model.impl.JavaInterfaceImpl and org.jboss.forge.roaster.model.source.JavaClassSource are in unnamed module of loader org.springframework.boot.loader.LaunchedURLClassLoader @5c7fa833)
java.util.concurrent.CompletionException: java.lang.ClassCastException: class org.jboss.forge.roaster.model.impl.JavaInterfaceImpl cannot be cast to class org.jboss.forge.roaster.model.source.JavaClassSource (org.jboss.forge.roaster.model.impl.JavaInterfaceImpl and org.jboss.forge.roaster.model.source.JavaClassSource are in unnamed module of loader org.springframework.boot.loader.LaunchedURLClassLoader @5c7fa833)
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1702)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1692)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
Caused by: java.lang.ClassCastException: class org.jboss.forge.roaster.model.impl.JavaInterfaceImpl cannot be cast to class org.jboss.forge.roaster.model.source.JavaClassSource (org.jboss.forge.roaster.model.impl.JavaInterfaceImpl and org.jboss.forge.roaster.model.source.JavaClassSource are in unnamed module of loader org.springframework.boot.loader.LaunchedURLClassLoader @5c7fa833)
	at com.github.cameltooling.lsp.internal.documentsymbol.DocumentSymbolJavaProcessor.getSymbolInformations(DocumentSymbolJavaProcessor.java:41)
	at com.github.cameltooling.lsp.internal.documentsymbol.DocumentSymbolProcessor.lambda$getDocumentSymbols$0(DocumentSymbolProcessor.java:42)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	... 6 more
```

**Comment by apupier** [comment]
@rafaeltuelho thanks for the report. Do you have any steps to reproduce? A specific file with which it occurs for instance?

**Comment by rafaeltuelho** [comment]
@apupier This seems to happen on a non-camel Java Project.
I'm not working with Camel at the moment. But when I open a Maven Java project I start to see annoying errors coming from Apache Camel Extension on my VSCode. 
Not sure If it's reproducible. I can share the whole log message if you will.

**Comment by apupier** [comment]
> I can share the whole log message if you will.

yes, please

**Comment by apupier** [comment]
> > I can share the whole log message if you will.
> 
> yes, please

hum ok, no need. I understood the issue. Will provide a fix soon

**Comment by apupier** [comment]
@rafaeltuelho awaiting for the next release, you can pick a snapshot containing the fix: https://download.jboss.org/jbosstools/vscode/snapshots/vscode-apache-camel/vscode-apache-camel-0.0.29-49.vsix

(no planned date for next release, unfortunately current days priorities are fighting to migrate to another CI)

**Comment by apupier** [comment]
fix is available on released version 0.0.29
